hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -392,7 +392,7 @@
     .q_o(${k}_sw_en)
   );
 
-  prim_mubi_pkg::mubi4_t ${k}_scanmode;
+  prim_mubi_pkg::mubi4_t [0:0] ${k}_scanmode;
```

Comment:
@tjaychen @msfschaffner are you okay with the change here in RTL to avoid FPV warnings? Thanks :)